examples/LaserWakefield/include/simulation_defines/param/species.param

```diff
@@ -34,7 +34,9 @@
 /*enable (1) or disable (0) electrons*/
 #define ENABLE_ELECTRONS 1
 /*enable (1) or disable (0) ions*/
+#ifndef ENABLE_IONS
```

Comment:
Even *this* is still a nostalgic remnant. Actually from the moment on when we will have three or more species in the simulation this `#define` will become rather unsuitable.